app/javascript/src/views/home.vue

```diff
@@ -1,14 +1,21 @@
 <template>
   <div class="home-container">
     <home-header />
-    <div
-      class="home-products-container"
-      v-if="products.length > 0"
-    >
-      <product
-        v-for="(product, index) in products"
-        :key="index"
-        :product="product"
+    <div v-if="category">
+      <p class="call-text">
+        Regala una de estas&nbsp;<span class="category-name">{{category.name}}</span>
```

Comment:
Expected 1 space after '{{', but not found          vue/mustache-interpolation-spacing<br>Expected 1 space before '}}', but not found         vue/mustache-interpolation-spacing